# HG changeset patch
# User Rik <rik@octave.org>
# Date 1423681101 28800
#      Wed Feb 11 10:58:21 2015 -0800
# Node ID c4728e4105b8ef9def4703a25454e0c539e5598a
# Parent  a4f6fbd1af77319392084d518bb74bf147f9420b
Remove duplicate assignment statement in cset c334e4af3a1b.

* __magick_read__.cc: Remove duplicate line "pix[GM_idx] = c;.

diff --git a/libinterp/dldfcn/__magick_read__.cc b/libinterp/dldfcn/__magick_read__.cc
--- a/libinterp/dldfcn/__magick_read__.cc
+++ b/libinterp/dldfcn/__magick_read__.cc
@@ -1106,17 +1106,16 @@ encode_uint_image (std::vector<Magick::I
             octave_idx_type GM_idx = 0;
             for (octave_idx_type col = 0; col < nCols; col++)
               {
                 for (octave_idx_type row = 0; row < nRows; row++)
                   {
                     const double grey = double (*img_fvec) / divisor;
                     Magick::Color c (grey, grey, grey);
                     pix[GM_idx] = c;
-                    pix[GM_idx] = c;
                     img_fvec++;
                     GM_idx += nCols;
                   }
                 GM_idx -= nCols * nRows - 1;
               }
             // Save changes to underlying image.
             m_img.syncPixels ();
             imvec.push_back (m_img);
